### ISSUE TITLE
z2m device not discovered after being deleted

### ISSUE THREAD

**Issue by designabc123** [Koenkk/zigbee2mqtt#19928]
### What happened?

Hello!  
Newbie here hoping you could help with an issue I wasn't able to fix for a week now. 

I recently discovered the benefits of zigbee devices being brought together and controlled in HA. I have a bunch of them from different companies, but I had issues from the start with my aqara roller blinds controller in HA. I have a Sonoff Zigbee Stick P as my coordinator and I originally started with ZHA as my HA platform. But there were some issues in ZHA with the aqara blinds motor, mainly some functionality that was unsupported. So I saw somewhere that z2m might provide better  compatibility. 

And to be fair, I was able to get access to more features with my device with z2m, but there were other major issues, like the state of how opened the blind is not updating. And again, I read somewhere that this must be caused by old Sonoff firmware. So I embarked in the strenuous journey of updating the firmware, and, after a few headaches, I managed to do it. 

After the firmware update, all my zigbee devices where still there, but my aqara blinds had the same problem. I thought that it might be because the device should be refreshed, thinking that I might get it working  properly if I delete and put it back in z2m. So I confidently went ahead and deleted the blinds, but when I wanted to pair them back into z2m, they were not being discovered. 

I had other devices in there that were working fine, I even bought an Aqara zigbee button to see if I can add it in, and that worked. So my issue is that I can't add the Aqara blinds back into z2m after being deleted. I put the blinds in pairing more, I turn z2m in pairing more too, but they don't get discovered. I've tried disconnecting my raspberry pi from power, disconnect the Sonoff stick from it too and, even though that fixed the issues for others that I read had a similar issue, nothing changed for me. 

Would anyone have any other suggestions? I really had big plans to get a decent zigbee network with lots of devices throughout the house, but I don't want to keep adding and configuring devices if I'll end up deleting or resetting them all in my attempt to get the blinds working. 

Many thanks in advance,
Liviu

### Zigbee2MQTT version

1.33.2

### Adapter firmware version

20230507

### Adapter

Sonoff Universal Zigbee Dongle Plus

**Comment by TermeHansen** [comment]
I just ran into this exact same issue, and also tried to stop docker and remove coordinator physically for more than 1min but still no luck (and no info in log). Any ideas other than changing channel and adding all devices again :(

Zigbee2MQTT version
    [1.36.1]

Coordinator type
    EmberZNet

Coordinator revision
    7.4.2 [GA]

**Comment by dietriro** [comment]
I have the same issue and the solution described above does not work for me either. Isn't there anyone who can help with this?

**Comment by dietriro** [comment]
I just found a solution which worked at least for me. Since there is no persistent data available on the server side (I run zigbee2mqtt in a docker and there are no files with information on the removed device on disk), I figured that the client device which tried to join the network was the issue. 

Removing the battery of the client device for 30 seconds did the trick and apparently reset something. Now the device was able to rejoin easily.
